boards/arm/frdm_k64f/frdm_k64f.dts

```diff
@@ -71,7 +71,7 @@
 		};
 		user_button_3: button_1 {
 			label = "User SW3";
-			gpios = <&gpioa 4 GPIO_INT_ACTIVE_LOW>;
+			gpios = <&gpioa 4 (GPIO_INT_ACTIVE_LOW | GPIO_PUD_PULL_UP)>;
```

Comment:
Is there a reason you didn't make the same change for the other button too?

Reply:
I tested the two buttons, the another button has the callback when releasing the button. I think that there is pull-up from the board's other ICs. Any way, I can do the update for another button too, it is  more clear and consistent. What do you think?

Reply:
Looking at the schematic, this pin (PTA4) has an external pullup on the board and shouldn't need to enable the internal pullup. The other pin (PTC6), however, does *not* have an external pullup and needs to enable the internal pullup.

The reason PTA4 is different on the board is that it is the NMI pin, but we've muxed it as a gpio.

Reply:
The R75 pull-up resistance is not soldered defaultly, I check three boards, all them are not sldered. That's why we need enable the internal pullup. Actually, the PTC6 (SW2) works well from my test, I think the reason is: U8 pulls up the PTC6.

Reply:
It turns out this depends on which revision of the board you have. R75 is populated on rev E, but DNP on rev C. Not sure about rev D, I don't have that schematic handy.